src/argilla/monitoring/asgi.py

```diff
@@ -101,7 +101,7 @@ async def cached_receive() -> Message:
         return self._receive
 
 
-class argillaLogHTTPMiddleware(BaseHTTPMiddleware):
+class ArgillaLogHTTPMiddleware(BaseHTTPMiddleware):
     """An standard starlette middleware that enables argilla logs for http prediction requests"""
 
     def __init__(
```

Comment:
I think these variables can be a bit more aligned with the pipeline monitors, wherever possible. I am explicitly talking about `log_interval`and `sample_rate`here.
https://github.com/argilla-io/argilla/pull/1905/files

Reply:
I was also thinking about giving preference of being able to set the prediction agent explicitly like `agent` in `rg.monitor`.